exercises/clock/test/test_clock.c

```diff
@@ -426,42 +426,8 @@ void test_subtract_more_than_two_days(void)
                                 minute_offset);
 }
 
-// these tests check equality.
-// Part of standard test collection
-// Included even though they don't make much sense in C
-
-// // trying it with XMACROS !
-#define EXPAND_EQUALITY_TEST_AS_TEST(test_name, hour1, minute1, hour2, minute2, expected_equality)\
-   void test_name(void)\
-   {\
-   test_time_expected_equality(hour1, minute1, hour2, minute2, expected_equality);\
-   }\
-
-// below expands to multiple routines...that are all designed to test the equality of 2 different
-// clock representations to see if they are the same or not
-// that 2 clocks are equal...
-// the fields are:
-//    test name, time1_hour, time1_minute, time2_hour, time2_minute, (time1 == time2)
-
-#define EQUALITY_TESTS(ENTRY) \
-   ENTRY(test_clocks_with_same_time,      15,   37,   15, 37, true)\
-   ENTRY(test_clocks_a_minute_apart,      15,   36,   15, 37, false)\
-   ENTRY(test_clocks_an_hour_apart,       15,   37,   14, 37, false)\
-   ENTRY(test_clocks_with_hour_overflow,  10,   37,   34, 37, true)\
-   ENTRY(test_clocks_with_hour_overflow_by_several_days,  3,   11,   99, 11, true)\
-   ENTRY(test_clocks_with_negative_hour,  22,   40,   -2, 40, true)\
-   ENTRY(test_clocks_with_negative_hour_that_wraps,  17,   3,   -31, 3, true)\
-   ENTRY(test_clocks_with_negative_hour_that_wraps_multiple_times,  13,   49, -83, 49, true)\
-   ENTRY(test_clocks_with_minute_overflow,  0,  1,   0, 1441, true)\
-   ENTRY(test_clocks_with_minute_overflow_by_several_days,  2,  2,   2, 4322, true)\
-   ENTRY(test_clocks_with_negative_minute,  2,  40,   3, -20, true)\
-   ENTRY(test_clocks_with_negative_minute_that_wraps,  4,  10,   5, -1490, true)\
-   ENTRY(test_clocks_with_negative_minute_that_wraps_multiple_times,  6,  15,   6, -4305, true)\
-   ENTRY(test_clocks_with_negative_hours_and_minutes,  7,  32,   -12, -268, true)\
-   ENTRY(test_clocks_with_negative_hours_and_minutes_that_wrap,  18,  7,   -54, -11513, true)\
-
-EQUALITY_TESTS(EXPAND_EQUALITY_TEST_AS_TEST)
-
+//Equality of two clock objects tests not implemented because C does not
+//have objects.
```

Comment:
Everyone OK with this?

Reply:
IIRC the tests compared the resulting strings had equal contents, rather than that any objects are equal... the former should be achievable in C?

Reply:
I'm not saying the tests cannot be written.  I'm saying that they don't add anything to the suite above.  In Java you would have to override the Equals() method to get these tests to pass.

Looking at this as a method of writing TDD: `Red, Green, Refactor` these should all be green the instant you run them.

Reply:
I think this is fine, but adding a note here is probably just going to confuse a user. How about we omit them but keep it to ourselves?